modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -414,6 +418,16 @@ public void processOpts() {
             additionalProperties.put(POD_AUTHORS, DEFAULT_POD_AUTHORS);
         }
 
+        if (additionalProperties.containsKey(USE_SPM_FILE_STRUCTURE) && (boolean)additionalProperties.get(USE_SPM_FILE_STRUCTURE)) {
+            setUseSPMFileStructure(convertPropertyToBooleanAndWriteBack(USE_SPM_FILE_STRUCTURE));
+            sourceFolder = "Sources";
```

Comment:
@aymanbagabas according to https://swift.org/package-manager/
`By convention, a target includes any source files located in the Sources/<target-name> directory.`
So I would suggest to change the structure to `Sources/ProjectName`.
What do you think?

Reply:
I think you're right, at some point, I dropped the projectName from the source folder somehow.